content/docs/12.5-vertex/02-install-palette-vertex/10-install-on-vmware.md

```diff
@@ -0,0 +1,202 @@
+
+<br />
+
+#### Environment Configuration
+
+
+|**Parameter**| **Description**|
+|:-------------|----------------|
+|**HTTPS Proxy**|Leave this blank unless you are using an HTTPS Proxy. This setting will be propagated to all PCG nodes and all of its cluster nodes. Example: ``https://USERNAME:PASSWORD@PROXYIP:PROXYPORT``.|
+|**HTTP Proxy**|Leave this blank unless you are using an HTTP Proxy. This setting will be propagated to all PCG nodes and all of its cluster nodes. Example: ``http://USERNAME:PASSWORD@PROXYIP:PROXYPORT``.|
+|**No Proxy**|The default is blank. You can add a comma-separated list of local network CIDR addresses, hostnames, and domain names that should be excluded from being a proxy. This setting will be propagated to all the nodes to bypass the proxy server. Example if you have a self-hosted environment: ``maas.company.com,10.10.0.0/16``.|
+|**Proxy CA Certificate Filepath**|The default is blank. You can provide the filepath of a CA certificate on the installer host. If provided, this CA certificate will be copied to each host in the PCG cluster during deployment. The provided path will be used on the PCG cluster hosts. Example: `/usr/local/share/ca-certificates/ca.crt`.|
+|**Pod CIDR**|Enter the CIDR pool IP that will be used to assign IP addresses to pods in the PCG cluster. The pod IP addresses should be unique and not overlap with any machine IPs in the environment.|
+|**Service IP Range**|Enter the IP address range that will be used to assign IP addresses to services in the PCG cluster. The service IP addresses should be unique and not overlap with any machine IPs in the environment.|
+
+<br />
+
+#### VMware vSphere Account Information
+
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+|**vSphere Endpoint** | vSphere endpoint: FQDN or IP address without a scheme - that is, without an IP protocol, such as ``https://``. Example: `vcenter.mycompany.com`.|
+|**vSphere Username** | vSphere account username.|
+|**vSphere Password**| vSphere account password.|
+|**Allow Insecure Connection** | Bypasses x509 verification. Enter `y` if using a vSphere instance with self-signed Transport Layer Security (TLS) certificates. Otherwise, enter `n`.|
+
+<br />
+
+#### VMware vSphere PCG Cluster Configuration
+
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+|**Datacenter**| |
+|**Folder** | |
+|**Fault Domain(s)** | Configure one or more fault domains by selecting values for these properties: Cluster, Network, Resource Pool, and Storage Type (Datastore or VM Storage Policy). |
+|**NTP Server(s)** | You can provide a list of Network Time Protocol (NTP) servers.  |
+|**SSH Public Key(s)** | Provide any public SSH keys you will use to access clusters in your project. |
+|**PCG cluster size** | Specify **1** or **3** nodes (HA). |
```

Comment:
This might be copy/paste info that doesn't belong. I'll double check. I think it doesn't apply since the installer creates a three-node EC cluster by default.